admin2.php

```diff
@@ -54,7 +54,7 @@
 					<div class = "logo2">

```

Comment:
Same images and graphics from admin1.php have been applied to admin2.php for consistency.